README.md

```diff
@@ -81,3 +82,16 @@ Additional files provided by WRF-CMake are licensed according to [LICENSE_CMAKE.
 ```
 Copyright 2018 M. Riechert and D. Meyer. Licensed under the MIT License.
 ```
+# Original README.md content
```

Comment:
I am not sure I like this here. It seems confusing. The other option may be to keep the file as is an prefix it or suffix it with something like `WRF-Make`... but I am not sure that is a good option either

Reply:
I think it's less confusing if it's in a separate file. Maybe naming it `README_WRF.md` and cross-linking to it from our README.md is fine?

Reply:
Too late now, but I wonder if future endeavors of this sort (i.e. new build systems) might be able to benefit from the use of git submodules. You would still probably have to fork the original repo to adjust and assimilate changes from upstream, but then that repo could either be a submodule within the new build system repo, or, you could make the new build system a submodule of the forked repo.

That could help to also clarify some licensing & tagging/releasing issues too.

As I said, I don't think this would be a good idea now, but something worth considering if there is ever a "next time."

Reply:
@zbeekman It's a tricky one, but yes, I agree it would at least make licensing etc. more clear. The main reason we haven't done it is to keep it simple if people want to modify/extend WRF for experiments. Dealing with two repositories in this case would mean more burden for them.